### PR TITLE
Support multiline env vars

### LINKED ISSUE #3972: FlowFuse UI for configuring Environment Variables in NodeRED instances truncates / corrupts large multi-line values
### Current Behavior

When trying to set environment variables containing secrets consisting of multi-line base 64 encoded data, both the UI options of pasting into an edit field or importing a .env file result in corruption of the value being imported. The edit field removes newlines and truncates the value. The import of a .env file containing enough multiline data results in multiple truncated environment variables being created. (e.g a file containing 1723 bytes in my test was split into two variables, the second with a random name).

### Expected Behavior

At least one of the FlowFuse UI options for setting environment variables should allow multi-line values of over 1k to be set without modification of the imported data, to support providing secrets that are large cryptographic keys.

### Steps To Reproduce

1. Create a file containing multi-line base-64 encoded data - preferably of over 1.7k in size (I haven't tested with smaller files, having encountered this problem using PEM format RSA keys)
2. In FlowFuse, select an Application and NodeRED instance, then navigate to Settings>Environment
3. Create a new environment variable using the UI and paste the contents of the test file into the "Value box.
4. Click "Save Settings"
5. Inspect the value of the environment variable, which is seen to to have no newline characters, and to be trucated
6. Modify the test file to create a multi-line .env file, and import it using the "Import .env" button.
7. In the Environment Variable UI verify that multiple trucated environment variables have been created (this may depend on the specific character sequence in the file)

### Environment

- FlowFuse version: 2.4.0 Docker Compose installation
- Node.js version: Default for Docker Compose (don't know how to find this info)
- npm version: Default for Docker Compose (don't know how to find this info)
- Platform/OS: Oracle Linux 9 VM
- Browser: Chrome

### Have you provided an initial effort estimate for this issue?

I am not a FlowFuse team member

### PR DESCRIPTION
**Pull request by Steve-Mcl**
closes #3972

## Description

TLDR;
Permit user to input multiline env vars by using `textarea` instead of standard `input` fields. Adjusts the view/styling to ensure faithful representation (using `whitespace: pre`) any new lines while constraining width/hieght limits (and adding auto scroll bars where necessary). Permits import of dotenv compatible data. 

DETAILS:
- Uses the FormView slot to show a `textArea` control 
  - The text area is styled with scoped CSS primarily as MVP (though should other areas of FF require TextArea fields, we should retro this
  - The specific styling is as follows:
     - readonly/un-editable fields get the `default` cursor to clarify what is/isnt editable
     - long strings in the readonly views are limited in both width and height and now show scrollbars are required.
 - multiline support has been added to both the view and the .env import. This is a close approximation of the `node` package `dotenv` in that multiline strings are wrapped in quotes, comments and empty lines are discarded.
 - frontend unit test updated to test the new capabilities of the import

 
##### template
image

##### non template
image

##### template
image

##### non template
image

#### [fix empty table layout bug when editing template env vars](https://github.com/FlowFuse/flowfuse/pull/4069/commits/e223b5e4b88afde36df08525a45f4b52a8232781)
This commit was an opportunity to fix this layout issue:
image

## Related Issue(s)

#3972

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
@joepavitt, when trying out, please use a file created by a regular macos text editor (so that is creates newlines in native OS format).

Here is some sample data: 

```bash
BASIC=basic

# prev line deliberately blank
AFTER_BLANK=after_blank
EMPTY=
SINGLE_QUOTES='single_line_single_line'
SINGLE_QUOTES_SPACED='    single quotes single line untrimmed    '
DOUBLE_QUOTES="double_quotes_single_line"
DOUBLE_QUOTES_SPACED="    double quotes single line untrimmed    "
NEWLINES_QUOTED="expand\nnew\nlines"
NEWLINES_SINGLED_QUOTED='expand\nnew\nlines'
DONT_EXPAND=dont\\nexpand\\nescpated\\nnewlines 
# COMMENTS=excludes comment that looks like a key-value pair
EQUAL_SIGNS=handles_equals=sign
JSON_UNQUOTED={"foo": "bar"}
JSON_QUOTED='{"foo": "bar"}'
JSON_MULTILINE='{
  "foo": "bar"
}'
TRIM_SPACE_FROM_UNQUOTED=    some spaced out string    
USERNAME=therealnerdybeast@example.tld
    SPACED_KEY = should be trimmed
MULTI_DOUBLE_QUOTED="THIS IS A
MULTILINE
STRING"
MULTI_SINGLE_QUOTED='THIS IS A
MULTILINE
STRING'
MULTI_WITH_BLANK_AND_COMMENT="SHOULD
KEEP EMPTY

AND KEEP LINE STARTING WITH #
# a line starting with a hash
WHEN IT IS QUOTED"
```

**Review by joepavitt — APPROVED** [review]
